_docs/getting-started/centos.md

```diff
@@ -8,6 +8,8 @@ We only support Anaconda packages at the moment. If you do not wish to use Anaco
 
 We build Linux packages without CUDA support, with CUDA 9.0 support, and with CUDA 8.0 support, for both Python 2.7 and Python 3.6. These packages are built on Ubuntu 16.04, but they will probably work on CentOS as well (if they do not, please tell us by creating an issue on our [Github page](https://github.com/caffe2/caffe2/issues)). To install Caffe2 with Anaconda, simply activate your desired conda environment and then run one of the following commands:
 
+> If your gcc version is older than 5 (you can run ```gcc --version``` to find out), then append '-gcc4.8' to the package name. For example, run `conda install -c caffe2 caffe2-gcc4.8` instead of what's below. (The command you run should always start with `conda install -c caffe2`)
```

Comment:
Nice. Can you also give an example with CUDA to be really thorough? For example `conda install -c caffe2 caffe2-cuda9.0-cudnn7-gcc4.8` as well? Otherwise looks good!